src/js/chartAuthoring.js

```diff
@@ -151,14 +210,43 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // Adds aria attributes that only make sense within the context of the overall chart authoring tool
     // Specifically, adds:
     // - an aria-controls attribute for the dataEntryPanel's form referencing the unique IDs of the pie and legend
+    // - an aria-controls attribute for the reset button referencing the dataEntryPanel form
     floe.chartAuthoring.addAriaConnections = function(that) {
-
         var legendId = fluid.allocateSimpleId(that.chartAuthoringInterface.pieChart.legend.locate("table")),
             pieId = fluid.allocateSimpleId(that.chartAuthoringInterface.pieChart.pie.locate("pie")),
-            totalId = fluid.allocateSimpleId(that.chartAuthoringInterface.dataEntryPanel.locate("totalValue"));
-
+            totalId = fluid.allocateSimpleId(that.chartAuthoringInterface.dataEntryPanel.locate("totalValue")),
+            dataEntryFormId = fluid.allocateSimpleId(that.chartAuthoringInterface.dataEntryPanel.locate("dataEntryForm"));
 
         that.chartAuthoringInterface.dataEntryPanel.locate("dataEntryForm").attr("aria-controls", legendId + " " + pieId + " " + totalId);
+        that.chartAuthoringInterface.locate("reset").attr("aria-controls", dataEntryFormId);
```

Comment:
I've added these. I think you're right that it's best to model indirect control relationships as well (the reset button is only technically controlling the form, but since the form controls pie/legend/total, it controls those too).